spec/definitions/boxes/definitions/base_ubuntu14.04_kvm/vbox.sh

```diff
@@ -0,0 +1,22 @@
+# Without libdbus virtualbox would not start automatically after compile
```

Comment:
not a stopper for the merge but I'm wondering if we need this since we are not running on vbox